api/provider/aws/task_create.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Create runs a Task using the specified Create Task Request. The Create Task Request
+// contains the name of the Task, the Environment ID, and the Deploy ID. The ECS Task Definition
+// family and version are derived from the Deploy ID.
```

Comment:
This isn't true. There isn't a logical mapping of deploy id -> task def family/verson

Reply:
I based the last sentence on this part of the code in `api-refactor`: https://github.com/quintilesims/layer0/blob/api-refactor/api/provider/aws/task_create.go#L22

Deploy ID is passed into `lookupDeployNameAndVersion()` and returns a Deploy Name and Deploy Version. Unless this code flow is now different?

Reply:
sorry, I was a little cart-before-the-horse with my comment. See https://github.com/quintilesims/layer0/pull/349/files#diff-f550f783d7a4fe285b3c373cc0eef78fR15. 

At this point it's a little bit in-flight, but I think we are going to end up doing something like:
```
deployID = randomHash
familyName = req.DeployName
revision/deploy version  = determined by aws
```

Reply:
If that is the case, then I feel that the [CreateTaskRequest](https://github.com/quintilesims/layer0/blob/api-refactor/common/models/create_task_request.go) model also needs to be updated to account for extracting out a req.DeployName since it currently does not and I don't see that reflected in the PR you linked to me.